tasks/user.yml

```diff
@@ -1,12 +0,0 @@
----
-
-- name: Add patroni group

```

Comment:
What if someone wants to run patroni not under `postgres` user?

Reply:
I think patroni use 99% user postgres.

Reply:
Revert user.yml and comment